Geometry/HcalSimData/data/HcalProdCuts/2021/v2/HcalProdCuts.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
+  <SpecParSection label="regions.xml" eval="true">
+    <SpecPar name="hcalProdCutsPar">
+      <PartSelector path="//CALO"/>
+      <PartSelector path="//MBAT"/>
+      <PartSelector path="//MBBT.*"/>
+      <PartSelector path="//VCAL"/>
+      <Parameter name="CMSCutsRegion" value="HcalRegion" eval="false"/>
+      <Parameter name="ProdCutsForElectrons" value="1*mm"/>
```

Comment:
Why was this line deleted?
```
<Parameter name="ProdCutsForProtons" value="1*mm"/>
```

Reply:
Actually, that line is in "v1" but not the old `HcalProdCuts.xml`. I wonder why.

Reply:
>Why was this line deleted?
<Parameter name="ProdCutsForProtons" value="1*mm"/>

Hmm that line was not deleted, the file which was used in 2021 scenario was `Geometry/HcalSimData/data/HcalProdCuts/HcalProdCuts.xml `, which does not contain it.
I noticed it indeed appear in v1 and not in the rest, but that v1 was unused.
This might or might not need to be addressed, but would say is independent from this PR, where we do not want to change the XMLs in use, but just rename the SpecPars (which... has an effect on the results :) ).

Reply:
@ghugo83 If you could follow up with this issue after this PR, that would be helpful. We should make sure the correct version of `HcalProdCuts.xml` is in use.